VolleyPaint/Assets/Scripts/BallBehaviour.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Blobcreate.Universal;
+using Blobcreate.ProjectileToolkit;
+
+public class BallBehaviour : ProjectileBehaviour
+{
+    public float radius = 4f;
+    public float centerForce = 10f;
+    public float forceUplit;
+    public LayerMask scanMask;
+    public LayerMask selfMask;
+    //public bool isCurveBall;
+
+    Collider[] result = new Collider[16];
+
+    protected override void OnLaunch()
+    {
+
+    }
+
+    protected override void Explosion(Collision collision)
```

Comment:
can I get comments explaining what this does?

Reply:
Explosion method will run when the ball hits  the targeted spot. 
It will get all the objects within range (on hit) and store them in Collider[] result.
And it will push them out by adding forces to the targets if they have rigidbody.